packages/server/graphql/public/mutations/verifyEmail.ts

```diff
@@ -16,14 +16,14 @@ const verifyEmail: MutationResolvers['verifyEmail'] = async (
   context
 ) => {
   const {dataLoader} = context
-  const r = await getRethink()
+  const pg = getKysely()
   const now = new Date()
-  const emailVerification = (await r
-    .table('EmailVerification')
-    .getAll(verificationToken, {index: 'token'})
-    .nth(0)
-    .default(null)
-    .run()) as EmailVerification
+  const emailVerification =
+    ((await pg
+      .selectFrom('EmailVerification')
+      .selectAll()
+      .where('token', '=', verificationToken)
+      .executeTakeFirst()) as EmailVerification) || null
```

Comment:
+1 no need to coerce to EmailVerification || null since kysely knows the return type